### ISSUE TITLE
Non-Redis Publications always use redis-oplog even with overridePublishFunction: false and Meteor.publish

### ISSUE THREAD

**Issue by evolross** [cult-of-coders/redis-oplog#240]
In our app, we don't want every publication to use `redis-oplog` by default because we have a couple publications and counters that get updated thousands of times per second and we want to poll those every few seconds (otherwise Blaze will freeze the entire UI/template while the updates are happening). 

So we set `overridePublishFunction: false` in the `redis-oplog` settings. We then used `Meteor.publishWithRedis` on the publications we want to use `redis-oplog`. So far so good. 

The problem is that all of the non-`redis-oplog` publications that use `Meteor.publish` still use `redis-oplog` reactivity by default. So for each one, we had to add `disableOplog: true, pollingIntervalMs: 3000` in order to get them to use polling like this:
```
Meteor.publish('eventParticipants', function(eventId) {
   check(eventId, Match.OneOf(String, null));

   return Events.find({_id: eventId}, {
      fields: {participantCount: 1},
      disableOplog: true,
      pollingIntervalMs: 3000
   });
});
```

If we have the `disable-oplog` package added and we have `overridePublishFunction: false` in the `redis-oplog` settings, shouldn’t that make all non `Meteor.publishWithRedis` publications poll by default and with the default polling time? This isn’t how it’s working for us. 

This should be easy enough to reproduce on any project, but if you need a repo, I'll create one with instructions to reproduce.

**Comment by theodorDiaconu** [comment]
It was a bug with redisOplog detecting pollingIntervalMs that is fixed in release/1.2.6 branch. You can check it out. I'm sure it's related.

**Comment by evolross** [comment]
Will test this. Being able to use `overridePublishFunction: true` and adding `disableOplog` and `pollingIntervalMs` to turn off reactivity and enable polling sounds great. I will update the docs to reflect this once I test it.

**Comment by evolross** [comment]
I tested this and it doesn't seem to be working for me. I did a test with my `redis-oplog-issues` repo that uses `redis-oplog@1.2.6` (https://github.com/evolross/redis-oplog-issues).

In settings.json, I have: `"overridePublishFunction": true`

I made the Scorecard publication like this:
```
Meteor.publish('scorecard', function tasksPublication() {
    return Scorecards.find({
      owner: this.userId 
    }, {
      disableOplog: true,
      pollingIntervalMs: 5000
   });
});
```
And in the `scorecards.toggle` Meteor Method I commented out the nested callback that updates the `Tasks`. So the Method is now just updating the Scorecard's `status` field.  This is what I get using two different browsers (Chrome and Safari).  The update to the Scorecard `status` pill should use polling, and it's reactively updating cross-browser:

![redisoplogpolling](https://user-images.githubusercontent.com/7980725/36002636-b8b50b74-0cdf-11e8-9522-3c3dab21bfc1.gif)

**Comment by theodorDiaconu** [comment]
@evolross that's happening because of optimistic ui. Try changing the database, so you can see that it's being polled!

**Comment by evolross** [comment]
So in the GIF, on the right-side browser (Safari) I’m clicking the “Toggle Scorecard Status” and you notice the left-side browser’s (Chrome) status poll updates immediately. That’s not optimistic UI if it’s behind triggered from a different browser is it? That’s why I used two different browsers. 

Looks very reactive to me.

**Comment by theodorDiaconu** [comment]
Yes because that's how optimistic ui works. It'll propagate the change to all active listeners. When you 2 instances listen to the same data, redis-oplog merges it (on the same server) and dispatches changes to all listeners. 

To check that polling actually works, change it in the db, and you will see changing in view with a delay.

**Comment by evolross** [comment]
Ah, got it.  I tested this successfully. I started two different servers pointing to the same database.  With...
```
disableOplog: true,
pollingIntervalMs: 5000
```
... clicking the "Toggle Scorecard Status" button changes it immediately in one browser (optimistic UI) and then polls in the other, as expected.

Without the above, they both reactively update immediately via Redis, as expected.

One last question/issue I noticed before I close this.  If I have the above polling code enabled, if I do an update directly in the database, both browsers eventually update because of polling.  If I _don't_ have the above code and do an update directly in the database, neither browser _ever_ updates until I refresh the browser/template.  

Is this as expected?
